recipes/spiders/tudo_gostoso.py

```diff
@@ -10,6 +9,8 @@ class TudoGostosoSpider(Spider):
         'https://www.tudogostoso.com.br/receitas-vegetarianas',
     ]
 
+    denied_words = ['ovo', 'ovos', 'porco']
```

Comment:
O que acha de fazer o seguinte com essa variavel

```suggestion
    denied_words = re.compile('.*(ovos?|porco|bovina).*', re.IGNORECASE)
```
considerando importar 're'

Reply:
acho que da pra adicionar peixe também se pá.

Reply:
Sim, adicionei também frango. :D